pages/use-case/human-resource.vue

```diff
@@ -0,0 +1,250 @@
+        <BlogCard
+          v-for="article in articles"
+          :key="article.slug"
+          class="col-lg-4 mt-3 mb-5"
+          :article="article"
+        />
+      </div>
+    </div>
+
+    <CallToActionSection
+      :content="{
+        heading: 'Ready to dive in?',
+        subHeading: 'Register with us Now.',
+        btnPrimary: 'Signup',
+      }"
+    />
+  </div>
+</template>
+
+<script>
+import getSiteMeta from '@/utils/getSiteMeta'
+import Hero from '../../components/hr-solution/Hero.vue'
+import StandOutFeatureSection from '../../components/competitors/StandOutFeatureSection.vue'
+import HelpingFeaturesForHr from '../../components/hr-solution/HelpingFeaturesForHr.vue'
+import CreateHrForms from '../../components/hr-solution/CreateHrForms.vue'
+import KeyBenefits from '../../components/hr-solution/KeyBenefits.vue'
+import SeamlessIntegration from '../../components/hr-solution/SeamlessIntegration.vue'
+import TemplateSection from '../../components/TemplateSection.vue'
+import BlogCard from '../../components/blog/BlogCard.vue'
+import CallToActionSection from '../../components/CallToActionSection.vue'
+
+export default {
+  components: {
+    Hero,
+    StandOutFeatureSection,
+    HelpingFeaturesForHr,
+    CreateHrForms,
+    KeyBenefits,
+    SeamlessIntegration,
+    TemplateSection,
+    BlogCard,
+    CallToActionSection,
+  },
+  async asyncData({ $content }) {
+    let articles = await $content('blog')
+      .where({
+        published: true,
+        featured: false,
+      })
+      .search('human resource')
+      .sortBy('createdAt', 'desc')
+      .fetch()
+
+    const randIndex = Math.floor(Math.random() * (articles.length - 3))
+    articles = articles.slice(randIndex, randIndex + 3)
+
+    return {
+      articles: articles,
+    }
+  },
+  computed: {
+    meta() {
+      const metaData = {
+        type: 'website',
+        url: 'https://formester.com/online-form-builder/',
+        title: 'Online Form Builders for powerful form-building',
+        description:
+          'Effective data collection and management for businesses and individuals alike. Transform your data-collection process & derive valuable insights!',
+        mainImage:
+          'https://formester.com/online-form-builder-hero-screenshot.png',
+        mainImageAlt: 'Form builder showing drag and drop functionality',
+        keywords: [
+          'Online Form',
+          'Online Form Builder',
+          'Online Form Builders for powerful form-building',
+          'Form Building',
+          'web forms productivity',
+          'form builder',
+          'formester',
+          'online forms',
+          'online web forms',
+          'formester web forms',
+          'form creator',
+          'form generator',
+          'online form',
+          'web form',
+          'online forms',
+          'web forms',
+          'create form',
+          'create forms',
+          'frequently asked questions',
+          'FAQ',
```

Comment:
It hasn't been provided yet. For now you can add few keywords like: human resources, human resource management, human resource forms, employee forms etc.